packages/flutter/test/animation/tween_test.dart

```diff
@@ -63,8 +63,8 @@ void main() {
     );
     final Matrix4 c = a.clone()..rotateZ(1.0);
     final Matrix4Tween rotationTween = new Matrix4Tween(begin: a, end: c);
-    expect(rotationTween.lerp(0.0), equals(a));
-    expect(rotationTween.lerp(1.0), equals(c));
+    expect(rotationTween.lerp(0.0).absoluteError(a), moreOrLessEquals(0.0));
+    expect(rotationTween.lerp(1.0).absoluteError(c), moreOrLessEquals(0.0));
```

Comment:
why does this differ on windows?

Reply:
There was some issues with floating point errors on Windows (differences on the magnitude of 1e-16 for individual entries in the matrix) which was causing the `equals` check to fail. I'm actually surprised the original worked consistently before, since testing floats for equality is just asking for trouble.

Reply:
It's really weird that Mac and Linux would be consistent here but Windows would not. I think this points to a bug somewhere. I don't think we should check this in without better understanding what's going on.

Maybe a compiler somewhere is ordering the multiplications differently or something?